Extension/package.json

```diff
@@ -315,13 +316,13 @@
           "description": "%c_cpp.configuration.configurationWarnings.description%",
           "scope": "resource"
         },
-        "C_Cpp.intelliSenseCachePath": {
+        "C_Cpp.intelliSense.cachePath": {
           "type": "string",
```

Comment:
Should this be cache.path and cache.size?

Reply:
It was previously cachePath and cacheSize in the OneNote page, and I had split them.  But, then I encountered a complication in SettingsTracker which would not be an issue with cachePath and cacheSize.  Does anyone feel strongly this should be one way or the other?